harvard-lts/mps-viewer
# No https

**Remove https from internal container**
* * *

# What does this Pull Request do?
We can't have internal https in a container for k8s, so this removes it, as well as curl from the container as the new health check is present. There is a relate PR for MPS-VIEWER-IF to remove the curl healthcheck and use the new node on with http, once this is merged and we can cut a new release.

# How should this be tested?

Build this branch, confirm viewer comes up not https (this image is running on k8s)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
